### ISSUE TITLE
Streamtip functionality not working?

### ISSUE THREAD

**Issue by jbzdarkid** [Gocnak/Botnak#80]
I've hooked up botnak to my local twitch page, and tested out donating to myself. However, botnak doesn't post these in chat, and upon restarting doesn't list them as "most recent tip". Furthermore, it doesn't tell me it is loading donations, nor that it has failed to load them.

Is there some setup I have to do to get these running? If so, it should probably be documented somewhere...

**Comment by Gocnak** [comment]
Yes there's some stuff you gotta do with defaults.ini, taking the Streamtip API ClientID and putting it for  DCID= and the access token for DCOAUTH=

It was going to be part of the new Settings GUI so I never got around to documenting it yet.

**Comment by jbzdarkid** [comment]
I've done both of those, tip notifications are still not present.

**Comment by jbzdarkid** [comment]
Still not working.

**Comment by Gocnak** [comment]
I'm keeping this issue open for a reminder to make the Settings GUI be helpful for setting this up.

Perhaps a wiki page on it for helping people out.
